packages/es-components/src/components/containers/modal/Modal.js

```diff
@@ -120,7 +120,7 @@ function Modal(props) {
 
   const shouldCloseOnOverlayClick = backdrop !== 'static' && backdrop;
 
-  return (
+  return show ? (
```

Comment:
I prefer the `show && (` syntax that doesn't require the `: null` at the end.

Reply:
🤷‍♂️ I like the explicitness.

Reply:
Super.